code/modules/vehicles/armored/som_tank.dm

```diff
@@ -0,0 +1,78 @@
+//3x4
+/obj/vehicle/sealed/armored/multitile/som_tank
+	name = "\improper Malleus hover tank"
+	desc = "A terrifying behemoth, the Malleus pattern hover tank is the SOM's main battle tank. Combining excellent mobility and formidable weaponry, it has earned a fearsome reputation among TerraGov forces that have faced it."
+	icon = 'icons/obj/armored/3x4/som_tank.dmi'
+	turret_icon = 'icons/obj/armored/3x4/som_tank_gun.dmi'
+	damage_icon_path = 'icons/obj/armored/3x4/tank_damage.dmi'
+	icon_state = "tank"
+	hitbox = /obj/hitbox/rectangle/som_tank
+	interior = /datum/interior/armored
+	minimap_icon_state = "tank"
+	required_entry_skill = SKILL_LARGE_VEHICLE_DEFAULT //leave as is or??
+	armored_flags = ARMORED_HAS_PRIMARY_WEAPON|ARMORED_HAS_SECONDARY_WEAPON|ARMORED_HAS_HEADLIGHTS
+	pass_flags = PASS_LOW_STRUCTURE|PASS_DEFENSIVE_STRUCTURE|PASS_FIRE
+	pixel_x = -65
+	pixel_y = -80
+	max_integrity = 900
+	soft_armor = list(MELEE = 65, BULLET = 75 , LASER = 80, ENERGY = 85, BOMB = 75, BIO = 100, FIRE = 100, ACID = 60)
+	hard_armor = list(MELEE = 0, BULLET = 20, LASER = 20, ENERGY = 20, BOMB = 0, BIO = 0, FIRE = 0, ACID = 0)
+	facing_modifiers = list(VEHICLE_FRONT_ARMOUR = 0.7, VEHICLE_SIDE_ARMOUR = 1, VEHICLE_BACK_ARMOUR = 1.5)
+	permitted_weapons = list(/obj/item/armored_weapon/volkite_cardanelle, /obj/item/armored_weapon/particle_lance, /obj/item/armored_weapon/coilgun, /obj/item/armored_weapon/secondary_mlrs)
+	permitted_mods = list(/obj/item/tank_module/overdrive, /obj/item/tank_module/ability/zoom) //revisit
+	max_occupants = 4
+	move_delay = 0.3 SECONDS
+	glide_size = 4.333 //todo: Fix glidesize for vehicles. It generates visibly choppy glide, possibly related to how vehicles use cooldown for movedelay
```

Comment:
given how slow they move i dont think theres a good way to fix it in a way that looks good, also the vehicle code should have stuff for this automatically also  remember desants need this too, and they can move independentently on top of the veh while gliding

Reply:
So this is as perfect a glide size I can get for this move delay. As I wrote in the note, (I bug reported it as well) something with how vehicle movement is calculated does not line up with how mob/client movement is calculated, so if I use the normal glide size calc, it gives me a value of like 5.8 or something which is the horrible steppy movement.

Desants are fine because mobs calc their glide every time they move, including when moved by a vehicle (which provides the override)